extensions/amp-video/0.1/flexible-bitrate.js

```diff
@@ -0,0 +1,259 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {childElement, childElementsByTag} from '../../../src/dom';
+import {dev, devAssert} from '../../../src/log';
+import {listen, listenOnce} from '../../../src/event-helper';
+import {toArray} from '../../../src/types';
+
+const TAG = 'amp-video';
+
+/** @const {!Object<string, number>} */
+const BITRATE_BY_EFFECTIVE_TYPE = {
+  'slow-2g': 50,
```

Comment:
Just for posterity, where are these values sourced from? Might be useful to add a comment since they seem arbitrary.

Reply:
Added a comment. Trial and error